code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -205,6 +205,24 @@ datum
 			reagent_state = LIQUID
 			color = "#B97A57"
 
+		fishwater
+			name = "Fish Water"
+			id = "fishwater"
+			description = "Smelly water from a fish tank. Gross!"
+			reagent_state = LIQUID
+			color = "#757547"
+
+			on_mob_life(var/mob/living/M as mob)
+				if(!M) M = holder.my_atom
+				M << "Oh god, why did you drink that?"

```

Comment:
Yeah, was just referring to the "why did you drink that" message; not the vomitting.